### ISSUE TITLE
Closing <p> tags without opening tags

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#528]
```
Please describe the issue:
In the HTML5 Slideshow(minimal version), there is a series of </p> tags in a 
list. However there are no opening tags for <p>.

Please provide any additional information below.
This is the source code:
<ul> 
  <li>[arrow keys] to go next and previous</p></li> 
  <li>[mouse scroll wheel] to go next and previous</p></li> 
  <li>[Ctrl or Command] + [+/-] to zoom in and out</p></li> 
  <li>[touch gestures] for mobile devices</p></li> 
</ul>
```

Original issue reported on code.google.com by `demoneau...@gmail.com` on 30 May 2011 at 4:59

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@gmail.com` on 1 Jun 2011 at 11:46
- Changed state: **Assigned**
- Added labels: **Priority-P3**, **Milestone-6**, **Slides**
- Removed labels: **Priority-P2**

**Comment by GoogleCodeExporter** [comment]
Original comment by `han...@google.com` on 2 Jun 2011 at 2:42
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in Revision: 8cf71ee6d4e3
```

Original comment by `han...@google.com` on 2 Jun 2011 at 9:06
- Changed state: **Fixed**
